packages/block-library/src/avatar/style.scss

```diff
@@ -1,4 +1,6 @@
 .wp-block-avatar {
+	box-sizing: border-box;
```

Comment:
This is a small nit, but can you add the comment 

> // This block has customizable padding, border-box makes that more predictable.

This keeps the [implementation consistent](https://github.com/WordPress/gutenberg/pull/43458/files) with other blocks and makes it easy to understand why this has been added. Otherwise, this PR looks good to go.

Reply:
Again, nothing too small, and thanks for the nitpicking! I addressed the missing comment. 👍